dockerfiles/openedx-codejail/Dockerfile

```diff
@@ -32,7 +32,9 @@ RUN git clone https://github.com/eduNEXT/codejailservice/ --branch main --depth
 
 # Install dependencies
 RUN source $CODEJAIL_VENV/bin/activate && pip install -r requirements/base.txt -r requirements/dev.txt && \
```

Comment:
We shouldn't need to install the dev requirements since this image won't be used for development. We will only need the `base.txt` requirements file to run this service.

Also, in order to reduce the overall image size we should use the `--no-cache-dir` flag for `pip install` so that the downloaded files are discarded after installing.